src/components/variable.jsx

```diff
@@ -107,14 +135,29 @@ const Variable = React.createClass({
                 {startDate && stopDate &&
                   <h3>Du <FormattedDate value={startDate} /> au <FormattedDate value={stopDate} />&nbsp;:</h3>
                 }
-                <pre><code>{formulas[date].content}</code></pre>
+                <pre><code>{this.renderLinkedFormulaVariables(formulas[date].content)}</code></pre>
               </div>
             )
           })
         }
       </div>
     )
   },
+  link(substring, quote) {
+    if (!substring.includes(" ") && this.props.variables[substring]) {
```

Comment:
According to the functions naming, I'd expect `renderLinkedFormulaVariables` to check that the substrings are valid variables, and `link` to just build a `Link`.